pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitFinalScanUseCase.java

```diff
@@ -135,7 +135,6 @@ public Optional<ScanError> processRGPFinalScan(ScanPayload scanPayload) {
                 if (StringUtils.isNotEmpty(rgpBloodKit.getKitLabel()) && kitLabel.equals(rgpBloodKit.getKitLabel())
                         || StringUtils.isEmpty(rgpBloodKit.getKitLabel())) {
                     rgpBloodKit.setKitLabel(kitLabel);
-                    rgpBloodKit.setDdpLabel(ddpLabel);
```

Comment:
I found this bug while testing again. This `rgpBloodKit.setDdpLabel(ddpLabel)` is assuming something that is not always true, even though it may not break all the time there is no need to setDdpLabel at this place. `rgpBloodKit` always has a ddpLabel